File: src/main/java/life/magpies/registry/commands/CommandList.java
package life.magpies.registry.commands;

import com.mojang.brigadier.context.CommandContext;
import net.fabricmc.fabric.api.command.v2.CommandRegistrationCallback;
import net.minecraft.server.command.ServerCommandSource;
import net.minecraft.text.Text;

import static net.minecraft.server.command.CommandManager.literal;

public class CommandList {
    public static void register() {
        CommandRegistrationCallback.EVENT.register((dispatcher, registryAccess, environment) -> {
            dispatcher.register(literal("list")
                    .requires(source -> source.hasPermissionLevel(1))
                    .executes(CommandList::execute));

        });
    }
    public static int execute(CommandContext<ServerCommandSource> ctx) {
        ServerCommandSource source = ctx.getSource();
        source.sendMessage(Text.literal("/kit -> 清除附近的掉落物 可选填半径最大30"));
        source.sendMessage(Text.literal("/day -> 白天"));
        source.sendMessage(Text.literal("/night -> 黑夜"));
        source.sendMessage(Text.literal("/sun -> 晴天"));
        source.sendMessage(Text.literal("/rain -> 雨天"));
        return 1;
    }
}

File: src/main/kotlin/life/magpies/registry/commands/CommandList.kt
package life.magpies.registry.commands

import com.mojang.brigadier.CommandDispatcher
import com.mojang.brigadier.context.CommandContext
import life.magpies.EasyCommand
import life.magpies.registry.config.ModConfig
import net.fabricmc.fabric.api.command.v2.CommandRegistrationCallback
import net.minecraft.command.CommandRegistryAccess
import net.minecraft.server.command.CommandManager
import net.minecraft.server.command.CommandManager.RegistrationEnvironment
import net.minecraft.server.command.ServerCommandSource
import net.minecraft.text.Text

object CommandList {
    fun register() {
        CommandRegistrationCallback.EVENT.register(CommandRegistrationCallback { dispatcher: CommandDispatcher<ServerCommandSource?>, registryAccess: CommandRegistryAccess?, environment: RegistrationEnvironment? ->
            dispatcher.register(CommandManager.literal("listEasy")
                .requires { source: ServerCommandSource ->
                    source.hasPermissionLevel(
                        ModConfig.list
                    )
                }
                .executes { ctx -> execute(ctx) }
            )
        })
    }

    private fun execute(ctx: CommandContext<ServerCommandSource>): Int {
        val source = ctx.source
        source.sendMessage(Text.translatable(("command." + EasyCommand.MOD_ID) + ".explain_kit"))
        source.sendMessage(Text.translatable(("command." + EasyCommand.MOD_ID) + ".explain_day"))
        source.sendMessage(Text.translatable(("command." + EasyCommand.MOD_ID) + ".explain_night"))
        source.sendMessage(Text.translatable(("command." + EasyCommand.MOD_ID) + ".explain_sun"))
        source.sendMessage(Text.translatable(("command." + EasyCommand.MOD_ID) + ".explain_rain"))
        source.sendMessage(Text.translatable(("command." + EasyCommand.MOD_ID) + ".explain_this"))
        source.sendMessage(Text.translatable(("command." + EasyCommand.MOD_ID) + ".explain_tpa"))
        source.sendMessage(Text.translatable(("command." + EasyCommand.MOD_ID) + ".explain_tpc"))
        source.sendMessage(Text.translatable(("command." + EasyCommand.MOD_ID) + ".explain_tpaccept"))
        source.sendMessage(Text.translatable(("command." + EasyCommand.MOD_ID) + ".explain_tpcetp"))
        return 1
    }
}